Fastfile

```diff
@@ -181,6 +182,16 @@ def _update_team_id_if_necessary
   end
 end
 
+def _update_code_signing_type_if_necessary
+  automatic_code_signing = ENV['FL_USE_AUTOMATIC_CODE_SIGNING']
+  if automatic_code_signing
+    UI.message("Updating code signing type, automatic: '#{automatic_code_signing}'")
```

Comment:
The message could read more Boolean like since that's what it is:
UI.message("Updating code signing to: '#{automatic_code_signing}'")

Reply:
Maybe we could follow the convention for boolean naming when we adding `is` 
`is_automatic_code_signing` ? Thoughts ?

Reply:
> Maybe we could follow the convention for boolean naming when we adding `is`
> `is_automatic_code_signing` ? Thoughts?

That's more iOS style - Ruby has no standards 😄

Reply:
Ahh. sure, let's stick to the convention we have then.